src/map.vue

```diff
@@ -2,65 +2,96 @@
-          maxZoom: 13
-          // continuousWorld: false,
-          // noWrap: true
-        })
-        .addTo(map);
+    // leaflet.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    leaflet
+      .tileLayer('https://maps.luftdaten.info/tiles/{z}/{x}/{y}.png', {
+        attribution:
+          'Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors',
+        maxZoom: 13
+        // continuousWorld: false,
+        // noWrap: true
+      })
+      .addTo(map);
 
-      let options = {
-        mouseover: () => {},
-        mouseout: () => {},
-        click: data => {
-          this.$emit("cell-selected", data);
-          document.getElementById("cell-info").style.display = "";
-          let row_counter = document.getElementsByClassName("cell_info_images");
-          for (var i = 0; i < row_counter.length; i++) {
-            row_counter[i].style.display = "none";
-          }
-          row_counter = document.getElementsByClassName("graph_on");
-          for (var i = 0; i < row_counter.length; i++) {
-            row_counter[i].style.display = "";
-          }
-          row_counter = document.getElementsByClassName("graph_off");
-          for (var i = 0; i < row_counter.length; i++) {
-            row_counter[i].style.display = "none";
-          }
+    const options = {
+      mouseover: () => {},
+      mouseout: () => {},
+      click: data => {
+        this.$emit('cell-selected', data);
+        document.getElementById('cell-info').style.display = '';
+        let row_counter = document.getElementsByClassName('cell_info_images');
+        for (var i = 0; i < row_counter.length; i++) {
+          row_counter[i].style.display = 'none';
+        }
+        row_counter = document.getElementsByClassName('graph_on');
+        for (var i = 0; i < row_counter.length; i++) {
+          row_counter[i].style.display = '';
+        }
+        row_counter = document.getElementsByClassName('graph_off');
+        for (var i = 0; i < row_counter.length; i++) {
+          row_counter[i].style.display = 'none';
         }
-      };
+      }
+    };
+
+    this.hexLayer = new leaflet.HexbinLayer(options).addTo(map);
+
+    // Leaflet-hash lets you to add dynamic URL hashes to web pages with Leaflet maps.
+    // You can easily link users to specific map views.
+    this.hash = new L.hash(map);
+
+    // Remove the hexlayer to on reload so that the position is updated
+    // map.setView or map.panTo called in Hash does not seem to update hexLayer
+    const refresh = throttle(() => {
+        map.removeLayer(this.hexLayer);
+        this.hexLayer.data([]);
+        this.hexLayer.addTo(map);
+        this.hexLayer.data(this.data);
+    }, 200); // hash has a default throttle of 100 ms
 
-      let hexLayer = new leaflet.HexbinLayer(options).addTo(map)
+    map.on('dragend', () => {
+      map.dragged = true; // don't reload when dragged
```

Comment:
QQ: Does this ever get reset back to `false`? Or once you've dragged once, it will always be `true`

Reply:
Yeah it gets reset https://github.com/CodeForAfrica/sensors.AFRICA-AQ-map/pull/17/files#diff-10329629e4e003e8671762aa141434b8R85

Reply:
Damn, [return-assign](https://eslint.org/docs/rules/no-return-assign)

Reply:
@kilemensi resolved